### PR TITLE
Yaml update for Gov Bruce Rauner (VAN-50386)

### PR DESCRIPTION
**Pull request by OdeclasV**
I noticed that selectors for this yaml were changed recently on the Governor's form. This solves that issue as well as edits other info pertaining to the topics.

### CODE REVIEW

File: members/M000303.yaml [hunk 1]
```diff
@@ -99,6 +101,8 @@ contact_form:
     - click_on:
       - value: Submit
         selector: div.buttonHolder input.btn-primary
+    - wait:
+      value: 3
```
Comment (tomdionesotes) [inline members/M000303.yaml]: needs indent + dash

File: members/M000303.yaml [hunk 2]
```diff
@@ -4,6 +4,8 @@ contact_form:
   action: ""
   steps:
     - visit: "http://www.mccain.senate.gov/public/index.cfm/contact-form"
+    - wait:
+      - value: 10
```
Comment (tomdionesotes) [inline members/M000303.yaml]: i would set this to something a bit lower unless a 10 second wait makes submissions succeed

File: members/state_il_gov_bruce_rauner.yaml [hunk 3]
```diff
@@ -6,115 +6,83 @@ contact_form:
...
+            - "17": "Senior Services"
+            - "18": "Taxes"
+            - "19": "Transportation & Infrastructure"
+            - "20": "Veterans & Military Affairs"
+            - "21": "Voting"
+            - "22": "Other Issues"
     - click_on:
         - value: Submit
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_SOIFeedbackSubmit"
+          selector: input[type='submit'][value='Submit']
```
Comment (tomdionesotes) [inline members/state_il_gov_bruce_rauner.yaml]: they removed the ID for the submit button on the form?

File: members/state_il_gov_bruce_rauner.yaml [hunk 4]
```diff
@@ -6,115 +6,83 @@ contact_form:
...
           options:
             - Support
             - Oppose
             - None
-        - name: topic
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Subject"
-          value: $TOPIC
+        - name: subject
+          selector: select#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Subject
+          value: $SUBJECT
```
Comment (tomdionesotes) [inline members/state_il_gov_bruce_rauner.yaml]: subject is an input, topic is for a dropdown with different mapped options. which does the form have?

File: members/state_il_gov_bruce_rauner.yaml (hunk 3 above)
Comment (OdeclasV) [reply members/state_il_gov_bruce_rauner.yaml]: Well not really. The computer generated ID is there, but Anna showed me this way to select the submit button a few weeks ago. I just think this is a safer for when the form is updated with new selectors. At least we know that we submit button is working. Thoughts?

Comment (annanhan) [reply members/state_il_gov_bruce_rauner.yaml]: in this case, if there is an ID on the button, we should use that. the more generic way , which you have, can be used if there is nothing else better. if these IDs are computer generated, the selectors for the other fields will fail, so changing this to what you have doesn't really get us anything. something better could be `submit#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_SOIFeedbackSubmit`

File: members/M000303.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/M000303.yaml]: These changes are for Mccain's yaml. It's here as a mistake. Please see the other PR

File: members/M000303.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/M000303.yaml]: These changes are for Mccain's yaml. It's here as a mistake. Please see the other PR

File: members/state_il_gov_bruce_rauner.yaml (hunk 3 above)
Comment (OdeclasV) [reply members/state_il_gov_bruce_rauner.yaml]: Understood. I'll change it and add the ID for the button

File: members/M000303.yaml (hunk 2 above)
Comment (tomdionesotes) [reply members/M000303.yaml]: these still need to be removed from this PR

File: members/M000303.yaml (hunk 1 above)
Comment (tomdionesotes) [reply members/M000303.yaml]: same as above - still need to take this edit out of this branch

Comment (OdeclasV) [reply members/M000303.yaml]: Sounds good. How do I take it out tho? Should I close this PR and create a new one instead?

**Pull request merged**